src/renderer/views/Settings/Settings.vue

```diff
@@ -1,34 +1,49 @@
 <template>
-  <div>
+  <div class="settings settingsPage">
```

Comment:
As these two hardcoded classes are just used for this element, could they be consolidated into a single class?